Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/BuiltInLinuxWebAppDeploymentProvider.ts

```diff
@@ -93,11 +98,15 @@ export class BuiltInLinuxWebAppDeploymentProvider extends AzureRmWebAppDeploymen
     }
 
     public async UpdateDeploymentStatus(isDeploymentSuccess: boolean) {
-        if(this.kuduServiceUtility) {
+        if(!this.kuduServiceUtility){
+            tl.debug('Kudu service utility not found.');
+            return;
+        }
+            tl.debug('LinuxWebAppDeploymentProvider120' + isDeploymentSuccess);
```

Comment:
What does it for 120?  Can you make it more readable and like a sentence for useful debug information?